spk/jackett/Makefile

```diff
@@ -1,19 +1,20 @@
 SPK_NAME = jackett
-SPK_VERS = 0.7.1483
-SPK_REV = 4
+SPK_VERS = 0.8.490
+SPK_REV = 3
```

Comment:
You probably want to increment the SPK_REV to 5, rather than decrement it to 3, or the package will not be proposed as update in the DSM package manager...

Reply:
done, the version is not considered by the package manager?